scripts/exgfs_wave_post_gridded_sbs.sh

```diff
@@ -221,22 +222,21 @@ source "${USHgfs}/preamble.sh"
 # 1.a.2 Loop over forecast time to generate post files
 # When executed side-by-side, serial mode (cfp when run after the fcst step)
 # Contingency for RERUN=YES
-  if [ "${RERUN-NO}" = "YES" ]; then
+  if [ "${RERUN:-NO}" = "YES" ]; then
     fhr=$((FHRUN + FHMIN_WAV))
     if [ $FHMAX_HF_WAV -gt 0 ] && [ $FHOUT_HF_WAV -gt 0 ] && [ $fhr -lt $FHMAX_HF_WAV ]; then
       FHINCG=$FHOUT_HF_WAV
     else
       FHINCG=$FHOUT_WAV
     fi
-    fhr=$((fhr + FHINCG))
+    fhr=$((FORECAST_HOUR + FHINCG))
   else
-    fhr=$FHMIN_WAV
+    fhr=$FORECAST_HOUR
```

Comment:
This section can just be removed. The forecast hour is provided and does not need to be calculated anymore.